channels/il.m3u

```diff
@@ -27,6 +27,8 @@ https://stream71.shidur.net/htvlive2/_definst_/smil:live2.smil/playlist.m3u8
 https://stream72.shidur.net/htvlive2/_definst_/smil:live2.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="IsraelParsTV.il" tvg-name="Israel Pars TV" tvg-country="IL" tvg-language="" tvg-logo="" group-title="",Israel Pars TV
 https://live.pars-israel.com/iptv/stream.m3u8
+#EXTINF:-1 tvg-id="i24NewsArabic.il" tvg-name="i24 News (Arabic)" tvg-country="IL" tvg-language="Arabic" tvg-logo="" group-title="News",i24 News (Arabic)
+https://iptv-all.lanesh4d0w.repl.co/israel/i24arabic
```

Comment:
The link still does not work via VLC.
![image](https://user-images.githubusercontent.com/7253922/120921760-dd665a80-c6cd-11eb-8b3a-46ab08e02d4c.png)

Reply:
Yes, this is apparently due to some things on their side, mine simply redirects to the resource.

Reply:
Do you also apply headers like referer and origin? Have you tried different user agents?

Reply:
Nothing like that, it's straight-forward. It is a link generated by their servers that I simply parse.